README.md

```diff
@@ -1,2 +1,82 @@
 # queries
 Hosts reusable log queries which are built into a single queries.json file.
+
+
+# Query database structure
+
+```yaml
+queries:
+  query1:  # that is query id (sorted by id)
+    name: foo  # required
+    regex: bar  # optional
+    pattern: aaa # optional (either regex or pattern required)
+    url: ... # str or list[str], issue links to lp, bz,... (optional)
+    category: str  # required, one of ...
+    tags: []  # lists of tags (optional)
+    suppress-graph: false  # used for e-r (optional)
+    # uncommon fields
+    multiline: true  # https://opendev.org/openstack/ansible-role-collect-logs/src/branch/master/vars/sova-patterns.yml#L47
```

Comment:
"multiline" here is just a flag to regex filter in Python, so it might be a suboption of "regex", in string patterns it doesn't make sense